### PR TITLE
Hs/sort networks

### LINKED ISSUE #3771: Sort network dropdown in the room directory alphabetically so that it's easier to find things
currently, the IRC networks seem to be sorted according to when they were added, which isn't the most helpful

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #3771.

### CODE REVIEW

File: src/components/views/directory/NetworkDropdown.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ export default class NetworkDropdown extends React.Component {
         // For our own HS, we can use the instance_ids given in the third party protocols
         // response to get the server to filter the room list by network for us.
         // We can't get thirdparty protocols for remote server yet though, so for those
-        // we can only show the default room list.
+        // we can only show the default room list.=
```
Comment (t3chguy) [inline src/components/views/directory/NetworkDropdown.js]: random equals?

Comment (Half-Shot) [inline src/components/views/directory/NetworkDropdown.js]: Appears to have sneaked in.

**Comment by uhoreg** [comment]
See also PR #4054

**Comment by ara4n** [comment]
oops, sorry, i ended up merging #4054 :/

**Pull request closed without merging**